geonode/qgis_server/views.py

```diff
@@ -740,3 +742,43 @@ def set_thumbnail(request, layername):
     }
     return HttpResponse(
         json.dumps(retval), content_type="application/json")
+
+@on_ogc_backend(qgis_server.BACKEND_PACKAGE)
+def download_qlr(request, layername):
+    """Download QLR file for a layer.
+
+    :param layername: The layer name in Geonode.
+    :type layername: basestring
+
+    :return: QLR file.
+    """
+    # execute only in qgis_server
+    if 'geonode.qgis_server' in settings.INSTALLED_APPS:
```

Comment:
Some comment as above, look at other views, we don't this kind of check as we are already in the QGIS Server app